packages/purpose-readmodel-writer/.env

```diff
@@ -0,0 +1,17 @@
+HOST=0.0.0.0
+PORT=3000
```

Comment:
Commenting here due to PR #424 
No need for these two as this is not an HTTP server

Reply:
Removed, thanks